drivers/modem/Kconfig.ublox-sara-n310

```diff
@@ -0,0 +1,30 @@
+# u-blox SARA N310 driver options
+
+# Copyright (c) 2019 Foundries.io
+# SPDX-License-Identifier: Apache-2.0
+
+config MODEM_UBLOX_SARA_N310
+	bool "Enable u-blox SARA N310 driver"
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	imply GPIO
+	help
+	  Choose this setting to enable u-blox SARA-N310
+	  driver.
+
+if MODEM_UBLOX_SARA_N310
+
+config MODEM_UBLOX_SARA_N310_INIT_PRIORITY
```

Comment:
It is defined this way in all modem drivers. It is given as a parameter in the device define, and because I don't know exactly why it needs to be 80 I took the advice in the "help" section and left it as is.